net-misc/ps3mediaserver/ps3mediaserver-1.90.1-r2.ebuild

```diff
@@ -0,0 +1,90 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit desktop xdg
+
+DESCRIPTION="DLNA compliant UPNP server for streaming media to Playstation 3"
+HOMEPAGE="https://github.com/ps3mediaserver/ps3mediaserver"
+SRC_URI="mirror://sourceforge/project/ps3mediaserver/pms-${PV}-generic-linux-unix.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="multiuser +transcode tsmuxer"
+
+BDEPEND="app-arch/unzip"
+RDEPEND=">=virtual/jre-1.8:*
+	media-libs/libmediainfo
+	media-libs/libzen
+	tsmuxer? ( media-video/tsmuxer )
+	transcode? ( media-video/mplayer[encode] )"
+
+S=${WORKDIR}/pms-${PV}
+PMS_HOME=/opt/${PN}
+
+src_prepare() {
```

Comment:
nit: all three "here documents" in this src_prepare are missing `die`. I know that this is already the case in the existing ebuild. But feel free to improve this ebuild while working on it. :)